### DISCUSSION TITLE
Adding a link in redactor doesn't always show the correct url

### DISCUSSION THREAD

**Discussion by riasvdv** [craftcms/cms#8458] [Ideas]
### Description
I'll try to describe the situation.

So we have a Craft 3 Multisite running with 6 different "sites"
- firstdomain.be - nl
- firstdomain.be - fr
- firstdomain.be - en
- seconddomain.be - nl
- seconddomain.be - fr
- seconddomain.be - en

So it's 2 domains with each 3 locales with "firstdomain.be" being the base domain.

1. Add a new page in the "parking-seconddomain.be - nl" locale
2. Add a link in the redactor field to another page
3. **Everything is correct and the link displays the correct url (in this case seconddomain.be/page#entry:xx)**
4. Close the page and reopen it
5. The redactor field shows the link as (firstdomain.be/page#entry:xx)

The url is still correct when you display the page on the frontend, even after re-saving, but this is causing some confusion with the content editors that think the links are going to the "wrong domain"

### Additional info

- Craft version: 3.0.0-beta.28
- PHP version: 7.1

**Comment by brandonkelly** [comment]
Issue here is that those reference tags aren’t storing the site that the entry should link to; it’s assumed that it will be the same site that the entry you are viewing is on.

**Comment by bartrylant** [comment]
I discovered the same, however, the links on the front-end are _wrong_. They all point to the domain they were created in, not the domain they refer to when you make the link.

**Comment by brandonkelly** [comment]
@bartrylant It will be whatever site you’re currently looking at. Again, reference tags are currently not site-aware, so it really doesn’t matter what site was selected when you made the link.
